projects/plugins/jetpack/modules/widget-visibility/widget-conditions.php

```diff
@@ -289,7 +289,7 @@ public static function widget_admin_setup() {
 			);
 
 			$widget_conditions_terms   = array();
-			$widget_conditions_terms[] = array( $taxonomy->name, __( 'All pages', 'jetpack' ) );
+			$widget_conditions_terms[] = array($taxonomy->name, get_taxonomy($taxonomy->name)->labels->all_items);
```

Comment:
Since `$taxonomy` in this case is a `WP_Taxonomy` instance, you may be able to pull the right info with `$taxonomy->labels->all_items` here:

```suggestion
			$widget_conditions_terms[] = array( $taxonomy->name, $taxonomy->labels->all_items );
```